feeadjuster/feeadjuster.py

```diff
@@ -13,6 +13,23 @@
 plugin.update_threshold = 0.05
 
 
+def get_adjusted_percentage(plugin: Plugin, scid: str):
+    """
+    For big channels, there may be a wide range where the liquidity is just okay.
+    Note: if big_enough_liquidity is greater than {total} * 2
+        then percentage is actually {our} / {total}, as it was before
+    """
+    channel = plugin.adj_balances[scid]
+    min_liquidity = min(channel["total"] / 2, int(plugin.big_enough_liquidity))
+    if channel["our"] < min_liquidity:
+        percentage = channel["our"] / min_liquidity / 2
+    elif channel["total"] - channel["our"] < min_liquidity:
+        percentage = (min_liquidity + channel["our"] - channel["total"]) / min_liquidity / 2 + 0.5
+    else:
+        percentage = 0.5
+    return percentage
```

Comment:
Maybe its just me, but I find this function rather hard to grasp (and test).

 - Can the actual `ours > min_liquidity` check which is implied by `if/else return 0.5` be somehow pulled out to line 113 right after the `is not significant_update(): continue` check? Only do this if you also think it makes things better, not worse.
 - If not, can the `return 0.5` be at least an early return?
 - You can add a local variable `theirs` to be `channel["total"] - channel["our"]` and use it to make things more readable.